packages/protocol-utils/src/transformer_utils.ts

```diff
@@ -87,6 +82,11 @@ export const fillQuoteTransformerDataEncoder = AbiEncoder.create([
             { name: 'fillSequence', type: 'uint8[]' },
             { name: 'fillAmount', type: 'uint256' },
             { name: 'refundReceiver', type: 'address' },
+            {
```

Comment:
shouldn't there be an accompanying change in FQT (solidity code)?

Reply:
yup forgot to add from /protocol was in protocol-utils